pkg/util/auth.go

```diff
@@ -0,0 +1,6 @@
+package util
+
+const (
+	// AuthenticationContextName will be used as a key to store authentication tokens in grpc call
+	AuthenticationContextName = "X-Auth"
```

Comment:
Out of curiosity, is "X-Auth" a wellknown header?

Reply:
Why not use "Authorization: Bearer [token]"

https://tools.ietf.org/html/rfc6750#section-2.1

Reply:
Also, can you move it to proto/header/header.go, like in this [PR](https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/46/files#diff-6989bdcc4293a974aef57ca42635665d)? This essentially is part of our api, not a util.

Reply:
1. moved to proto/header/header.go 2. Aligned with https://tools.ietf.org/html/rfc6750#section-2.1 standard